uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/rest/PortletsRESTController.java

```diff
@@ -103,6 +103,7 @@ public ModelAndView getPortlet(
         IPortletDefinition portletDef =
                 portletDefinitionRegistry.getPortletDefinitionByFname(fname);
         if (portletDef != null && ap.canRender(portletDef.getPortletDefinitionId().getStringId())) {
+            final IPerson person = personManager.getPerson(request);
```

Comment:
Why adding this line as it doesn't seem used ?

Reply:
good catch.  Leftover from a interim direction.  I'll remove it